src/editor/Editor.js

```diff
@@ -885,6 +887,18 @@ define(function (require, exports, module) {
         this._updateHiddenLines();
     };
 
+    Editor.prototype.mutualExcluse = function (f, changeList) {
```

Comment:
It makes sure that if a togetherjs event has triggered a change in the codemirror instance of this particular editor, then it shouldn't transmit a new event to all the connected clients.
I saw this in y-js's collaboration code.
But I think it should not be implemented as a prototype function to the editor, more like a global function.